### DISCUSSION TITLE
thoughts/%EB%8F%85%ED%9B%84%EA%B0%90/%EC%B2%A0%ED%95%99/2022-about-a-very-human-life/

### DISCUSSION THREAD

**Discussion by giscus** [rlaisqls/blog-comments#6] [Announcements]
# thoughts/%EB%8F%85%ED%9B%84%EA%B0%90/%EB%8F%85%ED%9B%84%EA%B0%90-2022/about-a-very-human-life/

https://blog.rlaisqls.site/thoughts/%EB%8F%85%ED%9B%84%EA%B0%90/%EB%8F%85%ED%9B%84%EA%B0%90-2022/about-a-very-human-life/

**Comment by jisoung0405** [comment]
전위적인 이상을 쫒는 것도 좋지만 존재함에 중점을 두고 사는 것도 좋은 방법일거 같아요~

**Reply by rlaisqls** [reply]
존재함에 중점을 두고 살기 위해선 쇼펜하우어가 말한 것 처럼 내면의 세계를 성장시키고 집중할 수 있는 힘을 길러야할 것 같네요.. 글 읽어주셔서 감사합니다 :)
